target/linux/ramips/image/mt7621.mk

```diff
@@ -1387,6 +1387,33 @@ define Device/xiaomi_mi-router-ac2100
 endef
 TARGET_DEVICES += xiaomi_mi-router-ac2100
 
+define Device/xiaomi_mi-router-cr6606
+  $(Device/dsa-migration)
+  $(Device/uimage-lzma-loader)
+  DEVICE_VENDOR := Xiaomi
+  DEVICE_MODEL := Mi Router CR6606
+  DEVICE_ALT0_VENDOR := Xiaomi
+  DEVICE_ALT0_MODEL := Mi Router CR6606
+  DEVICE_ALT0_VARIANT := China Unicom
+  DEVICE_ALT1_VENDOR := Xiaomi
+  DEVICE_ALT1_MODEL := Mi Router CR6608
+  DEVICE_ALT1_VARIANT := China Mobile
+  DEVICE_ALT2_VENDOR := Xiaomi
+  DEVICE_ALT2_MODEL := Mi Router CR6609
+  DEVICE_ALT2_VARIANT := China Telecom
```

Comment:
Consider dropping the *VARIANT variables here, as they don't look like they are used "correctly" (i.e. they don't really  help here at the moment).

Reply:
Basically, CR6608 and CR6609 both only have carrier customized variants for the time being. I'm curious whether Xiaomi would release a general variant like CR6606, so I tried to use it as an addition.
After some thought, I believe it would be nonsense in the build system. Dropped all the variants for now.